interface/wx/event.h

```diff
@@ -3318,6 +3318,62 @@ class wxDisplayChangedEvent : public wxEvent
 };
 
 
+
+/**
+    @class wxDPIChangedEvent
+
+    Event sent when the resolution (measured in dots-per-inch, or DPI) of the
```

Comment:
This may be just nitpicking, but I would not a word resolution here. I would say that when speakign about displays, resolution is something like 1920x1080 (i.e., the number of pixels a display has) and this does not change with a DPI change. Microsoft uses [Display Scale Factor](https://docs.microsoft.com/en-us/windows/win32/hidpi/high-dpi-desktop-application-development-on-windows), others use pixel density...